torch/csrc/utils/throughput_benchmark-inl.h

```diff
@@ -97,7 +97,8 @@ BenchmarkExecutionStats BenchmarkHelper<Input, Output, Model>::benchmark(
   using TimePoint = std::chrono::time_point<Clock>;
   TimePoint start_time;
 
-  std::unique_ptr<torch::autograd::profiler::RecordProfile> profiler_guard;
+  typedef torch::autograd::profiler::RecordProfile RecordProfile;
```

Comment:
Nit (and consider moving it to the rest of `using` block on lines 96-97)
```suggestion
  using RecordProfile = torch::autograd::profiler::RecordProfile;
```